src/Symfony/Bridge/Doctrine/Tests/Form/Type/EntityTypeTest.php

```diff
@@ -36,6 +36,7 @@
 
 class EntityTypeTest extends TypeTestCase
 {
+    const ENTITY_TYPE_CLASS = 'Symfony\Bridge\Doctrine\Form\Type\EntityType';
```

Comment:
:-1: for this change, it will complicate merges and is a personal preference "enhancement"